src/components/charts/chartTheme/theme-common.ts

```diff
@@ -0,0 +1,269 @@
+import { ChartTheme } from '@patternfly/react-charts';
+
+const defaultTheme = ChartTheme.light.blue;
+
+// Theme
+export default props => {
+  const {
+    // COLOR_AXIS_FILL,
```

Comment:
Should we keep those?

Reply:
I was keeping for a reference, but will clean this up a bit.